src/client/src/components/pages/participants/prisonerDetails/index.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from "react";
+import {
+  state as initialState,
+  fields as fieldSet,
+} from "./staticData";
+import Form from "./../../../abstract/Form";
+import Footer from '../../../abstract/footer';
+import axios from "axios";
+import swal from "sweetalert2";
+import contextHoc from './../../../abstract/HOC/contextHoc';
+
+class PresionDetails extends Component {
```

Comment:
This should be PrisonDetails!

```suggestion
class PrisonDetails extends Component {
```

Reply:
yeah you are right